journey-map/public/render.mjs

```diff
@@ -240,36 +270,57 @@ const calcOrphanStates = (journeyMap) => {
     return Object.keys(journeyMap).filter(state => !uniqueTargetedStates.includes(state));
 };
 
-export const render = (selectedJourney, journeyMaps, nestedJourneys, formData = new FormData()) => {
-    // Copy to avoid mutating the input
-    const journeyMapCopy = JSON.parse(JSON.stringify(journeyMaps[selectedJourney]));
+const getNestedJourneyStates = (nestedJourney) => ({
+    ...nestedJourney.nestedJourneyStates,
+    // Create an entry state for each entry event
+    ...Object.fromEntries(
+        Object.entries(nestedJourney.entryEvents)
+            .map(([event, def]) => [
+                `entry_${event}`.toUpperCase(),
+                {
+                    entryEvent: event,
+                    events: { [event]: def }
+                }
+            ]),
+    ),
```

Comment:
Nice! This looks way better and more helpful with the different entry events separated